packages/jsActions/nanoflow-actions-native/src/local-storage/ClearCachedSessionData.ts

```diff
@@ -0,0 +1,19 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+/**
+ * @returns {Promise.<void>}
+ */
+export async function clearCachedSessionData(): Promise<void> {
+    // BEGIN USER CODE
+    if (mx.session && mx.session.clearCachedSessionData === undefined) {
+        throw new Error("'JS action 'Clear cached session data' is not supported prior to Mendix client v9.14");
```

Comment:
Returning a rejected Promise follows conventions better. Im not sure whether this functionally makes a difference in the Nanoflow engine. What do you think? Here's an [example in the widgets-resources](https://github.com/mendix/widgets-resources/blob/master/packages/jsActions/nanoflow-actions-native/src/local-storage/GetStorageItemObject.ts#L20).

Reply:
Yeah, indeed this is better! ... will update!